hg-fast-export.py

```diff
@@ -122,6 +127,27 @@ def get_author(logmessage,committer,authors):
       return r
   return committer
 
+def generate_git_lfs_pointer(filename):
+  size = 0
+  hasher = hashlib.sha256()
+  with open(filename, "rb") as bfile:
+    buf = bfile.read(1024*1024*1024)
+    while buf:
+      hasher.update(buf)
+      size = size + len(buf)
+      buf = bfile.read(1024*1024*1024)
+  digest = hasher.hexdigest()
+  try:
+    os.makedirs("lfs/objects/%s/%s" % (digest[0:2],digest[2:4]))
+  except:
+    pass
+
```

Comment:
In this case, my intention where to output a message with step to make sure the file is downloaded in the cache and stop the process. Otherwise, I do think that making an automatic fetch may fail as you may be required to input a username and password and to my understanding the stdout is piped to git-fast-import.

But If we may know that username and password is not required to download the large files. Then it should be possible to download them automatically. Otherwise, I will stick to message and exit.